scripts/infra

```diff
@@ -71,8 +45,10 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
                 BATCH_JOB_NAME_REVISION=$(docker-compose run --rm update-job-defs pfb-analysis-run-job.json "${PFB_AWS_ECR_ENDPOINT}/pfb-analysis:${GIT_COMMIT}")
                 popd
+                terraform init \
```

Comment:
In the other Terraform 0.9 upgrades, we explicitly removed the `.terraform` directory before executing `terraform init`.